### PR TITLE
Support offline node contract notification on `mainnet`

### PR DESCRIPTION
**Pull request by Mahmoud-Emad**
This is open to be attached to the `mainnet` as a patch release.

### Support links in the notification pop-up

> - image
> - image

### The original issue description

> #### Description

> - Added a notification helper that will notify the user every 15 minutes if there are deployments on an offline node.
> - Updated the toast warning background, icon color, and text color in dark/light mode.   

> #### Related Issues

> - https://github.com/threefoldtech/tfgrid-sdk-ts/issues/1220

> - https://github.com/threefoldtech/tfgrid-sdk-ts/issues/1283 

> #### Screenshots

> - image

### CODE REVIEW

**Review by amiraabouhadid — CHANGES_REQUESTED** [review]
Please adjust the notification colors. I could not even read what is written in them. 

image

**Comment by Mahmoud-Emad** [comment]
@amiraabouhadid try to clear your cache keda

**Comment by 0oM4R** [comment]
image
working fine with me, I think it will be better to close that toast by clicking on the contracts link, as the user will be redirected the contracts list page.

File: packages/grid_client/src/clients/tf-grid/contracts.ts [hunk 1]
```diff
@@ -9,12 +10,67 @@ import { Decimal } from "decimal.js";
...
+
+export interface GqlContracts {
+  nameContracts: GqlNameContract[];
+  nodeContracts: GqlNodeContract[];
+  rentContracts: GqlRentContract[];
+}
+
+export interface GqlConsumption {
+  contracts: GqlContracts;
+  billReports: GqlContractBillReports[];
```
Comment (AhmedHanafy725) [inline packages/grid_client/src/clients/tf-grid/contracts.ts:63]:
```suggestion
  contractBillReports: GqlContractBillReports[];
```

File: packages/grid_client/src/clients/tf-grid/contracts.ts [hunk 2]
```diff
@@ -107,23 +164,25 @@ class TFContracts extends Contracts {
           }`;
     try {
       const response = await gqlClient.query(body, { contractId: options.id });
-      const billReports = response["data"]["contractBillReports"];
+      const gqlConsumption: GqlConsumption = response["data"] as GqlConsumption;
+      const billReports = gqlConsumption.billReports;
```
Comment (AhmedHanafy725) [inline packages/grid_client/src/clients/tf-grid/contracts.ts:168]:
```suggestion
      const billReports = gqlConsumption.contractBillReports;
```

**Review by AhmedHanafy725 — CHANGES_REQUESTED** [review]

**Comment by Mahmoud-Emad** [comment]
> image working fine with me, I think it will be better to close that toast by clicking on the contracts link, as the user will be redirected the contracts list page.

Done :)

**Review by 0oM4R — APPROVED** [review]
Great job thank you!

**Review by AhmedHanafy725 — APPROVED** [review]

**Comment by AlaaElattar** [comment]
- worked fine with me.

**Review by AlaaElattar — APPROVED** [review]

**Review by amiraabouhadid — APPROVED** [review]

**Pull request merged**